### ISSUE TITLE
RPLIDAR S2

### ISSUE THREAD

**Issue by ElbyB** [DanielKalicki/RPLidar_Arduino#2]
Hi,

Is it possible to use this library with the S2? 

Thanks!

**Comment by DanielKalicki** [comment]
I based my code on rplidar_sdk 1.12, this version did not supported S2:
https://github.com/Slamtec/rplidar_sdk/tree/release/v1.12.0

I check the commits history of the Slamtec rplidar and many changes were added to the library (i would even say that the original code was rework significantly) with the S2 support, so i dont expect my library to work out of the box. 
https://github.com/Slamtec/rplidar_sdk/commit/e3a572b0f24c035f4d08fff747f173fc6f0e9850#diff-218b399e180a7c9543a9b5b4f364e538c7d462b244c40e6f68b787baafe0510b

From the RPlidar S2 documentation I dont see many changes (versus A1 on which I tested) - the communication interface seems the same. You would need to test the code with your S2 lidar.
